packaging/_parser.py

```diff
@@ -0,0 +1,204 @@
+from ast import literal_eval
+from typing import Any, List, NamedTuple, Tuple, Union
+
+from ._tokenizer import Tokenizer
+
+
+class Node:
+    def __init__(self, value: str) -> None:
+        self.value = value
+
+    def __str__(self) -> str:
+        return str(self.value)
```

Comment:
```suggestion
        return self.value
```

Since `self.value` is always str